src/code/z_message_PAL.c

```diff
@@ -1837,7 +1836,7 @@ void Message_StartOcarina(PlayState* play, u16 ocarinaActionId) {
         msgCtx->msgMode = MSGMODE_TEXT_CONTINUING;
     }
     msgCtx->textboxColorAlphaCurrent = msgCtx->textboxColorAlphaTarget;
-    if (noStop == false) {
+    if (!noStop) {
```

Comment:
i wish this was named `continueMessage` or something, not a fan of the double negative

Reply:
The `Stop` here was more related to the "doAction" stop instead of the verb "stop", so I changed the variable to represent that better